.github/workflows/check_size.yaml

```diff
@@ -29,7 +29,20 @@ jobs:
         run: |
           python check_size.py
 
+      - name: Check github token
+        id: checktoken
+        shell: bash
+        run: |
+          if [ "${PRIVATE_GITHUB_TOKEN}" == "" ]; then
+            echo "::set-output name=token_exists::false"
+            echo "token_exists::false"
+          else
+            echo "::set-output name=token_exists::true"
+            echo "token_exists::true"
+          fi
```

Comment:
I would have liked a shorter version. I wasn't able to find one though 🤷

Reply:
Nice!